### ISSUE TITLE
Prgogramming Code markdown in Mindmap

### ISSUE THREAD

**Issue by moritzWa** [xmindltd/xmind#344]
As a **Software Developer** It would be great to use x mind to **view code snippets** and to memorize syntax and group types of problems. 

Software Engineering is a very visual task and mind maps are great to memorize and to learn visual knowledge.

Editing a branch I would like to be able to paste a code-block and have it recognized as one and then formatted in a fond like: _Source Code Pro, Hack or Monaco_ .

`Notion.io` has this feature.

If someone reads this Issue and the feature request gets declined please message me, so we can team up and build this feature into a fork. Currently, I use two pieces of software for note taking. X mind for theoretical concepts formulated in human language and Evernote for programming.

**Comment by salexfarm** [comment]
I agree. Another feature that would be great is a flashcard implementation (spaced repetition) with the ability to recall redacted portions by typing out what you have in the outliner or map.
